Automattic/jetpack
# My Jetpack: Add CRM interstitial page

Fixes https://github.com/Automattic/jetpack/issues/22889
Fixes #22842

#### Changes proposed in this Pull Request:

* My Jetpack: Add CRM interstitial page

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






With Storybook
* Take a look at the Product detail and interstitial components

CRM Card | Interstitial
------|-------
<img width="915" alt="image" src="https://user-images.githubusercontent.com/77539/154063423-d2bd2032-ea40-4b45-bf93-0266e484369a.png"> | <img width="1033" alt="image" src="https://user-images.githubusercontent.com/77539/154064192-29364174-6914-4bac-b367-33d876468c27.png">

* Go to My Jetpack
* Click on `Add CRM` link
* Confirm you see the interstitial page

<img width="1160" alt="image" src="https://user-images.githubusercontent.com/77539/154065511-6b75e4f9-69fb-42b3-9e65-432a42b1f7fd.png">
